src/main/kotlin/no/nav/klage/oppgave/api/view/VedtakView.kt

```diff
@@ -1,16 +1,15 @@
 package no.nav.klage.oppgave.api.view
 
 import no.nav.klage.oppgave.domain.klage.PartId
-import no.nav.klage.oppgave.domain.kodeverk.Hjemmel
-import no.nav.klage.oppgave.domain.kodeverk.Utfall
 import java.time.LocalDateTime
 import java.util.*
 
 data class VedtakView(
     val id: UUID,
-    val utfall: Utfall? = null,
-    val hjemler: MutableSet<Hjemmel> = mutableSetOf(),
-    val brevMottakere: MutableSet<PartId> = mutableSetOf(),
+    val utfall: String? = null,
+    val grunn: String? = null,
+    val hjemler: Set<String> = setOf(),
+    val brevMottakere: Set<PartId> = setOf(), //TODO: Er ikke noe happy med at vi eksponerer en Entity her, burde mappet det til noe annet
```

Comment:
Enig i at det ikke nødvendigvis er så bra. Her har vi vel ikke så tydelige planer for hva det skal brukes til? Denne er vel mest en placeholder før vi finner noe å bruke den til i GUI.

Reply:
Ok, skjønner. Da er jeg happy jeg!